rostime/include/ros/duration.h

```diff
@@ -120,6 +120,7 @@ class ROSTIME_DECL Duration : public DurationBase<Duration>
   explicit Duration(const Rate&);
   /**
    * \brief sleep for the amount of time specified by this Duration.  If a signal interrupts the sleep, resleeps for the time remaining.
+   * @return True if the desired sleep duration was met, false otherwise.
```

Comment:
Should the comments also mention that the function can return false if the sleep was interrupted (`g_stopped` became true)?